src/main/groovy/life/qbic/datasets/parsers/DataParserException.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.datasets.parsers
+
+/**
+ * Thrown to indicate that a dataset could not be parsed. 
+ * <p>The exception provides several types of information:
+ * <ul>
+ * <li>a string describing the exception. This is used as the Java exception message.</li>
+ * <li>the causal relationship, if any for this <code>DataParserException</code></li>
+ * </ul>
+ *
+ * @since 1.7.0
+ */
+class DataParserException extends RuntimeException {
+
+    DataParserException() {
+        super()
+    }
+
+    DataParserException(String message) {
```

Comment:
```suggestion
    /**
     * Constructs a <code>DataParserException</code> with the specified detail message.
     * @param message the detail message
     * @since 1.7.0
     */
    DataParserException(String message) {
```